### DISCUSSION TITLE
SSH Tunnel (Not ssh-remote)

### DISCUSSION THREAD

**Discussion by Kirbyrawr** [gitpod-io/openvscode-server#252] [Q&A] [answered]
Hi, i successfully started the server in a Raspberry Pi and tested locally, it works.

I'm now trying to connect to the server with an iPad using SSH Port Forwarding but the url gives a timeout with the token. If i dont use the token it gives a forbidden (this is ok).

I'm wondering if i need to do anything in order to ssh tunnel to it, maybe use nginx?

Thanks!

**Comment by jeanp413** [comment, marked as answer]
@Kirbyrawr you need to setup a reverse proxy (nginx or apache2) to access it using https, you can check this discussion that contains an apache2 config to setup a reverse proxy https://github.com/gitpod-io/openvscode-server/discussions/96#discussioncomment-1404354
